.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.6'
+        - '2.7'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+    - name: Run tests
+      run: bundle exec rake
+  build:
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up environment
+      run: |
+        echo "TRAVIS_PULL_REQUEST=${{ github.base_ref == '' && 'false' || 'true' }}" >> $GITHUB_ENV
+    - name: Set up registry credentials
+      if: ${{ github.base_ref == '' }}
```

Comment:
I'm guessing this is the "if not a PR" logic, right?

Reply:
Yes, that's the closest thing I could find for it.